app.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'lib/database_connection'
+require_relative 'lib/item_repository'
+require_relative 'lib/order_repository'
+require_relative 'lib/menu_result'
+require_relative 'lib/menu'
+
+DatabaseConnection.connect('shop_manager')
+
+
+menu = Menu.new(Kernel)
+menu_result = MenuResult.new(Kernel)
```

Comment:
I might suggest renaming the class `MenuResult` to something like `Application` or `ConsoleApplication`, as it provides a wrapper for other components of the program (e.g the Repository classes), but doesn't necessarily do anything related to showing the user menu / getting for it, as this is already handled by the `Menu` class